frontend/src/dataUtilities.ts

```diff
@@ -427,3 +459,22 @@ export const transformAccountsData = (data: AccountsGetResponse) => {
     return obj;
   });
 };
+
+interface PaymentData {
+  error: null;
+  payment: PaymentInitiationPaymentGetResponse;
+}
+
+export const transformPaymentData = (data: PaymentData) => {
+  return [
```

Comment:
this can be a shorthand function no need for a function body and a return

```suggestion
export const transformPaymentData = (data: PaymentData) => [
```

Reply:
darnit...yes, i keep forgetting about this shorthand function...